src/components/StatusPanel.vue

```diff
@@ -32,3 +52,49 @@ export default {
   }
 };
 </script>
+<style>
+#status {
```

Comment:
Please move this in style.scss to be compiled during the build process

Reply:
Just the `#status` rule, or the whole style sheet?

Reply:
the whole sheet

Reply:
53b8d6bc36c692d5896a62dcdc87a0c6e11ed93d